about/process/ppa.rst

```diff
@@ -1,32 +1,33 @@
+UBports Debian repository
+=========================
+
 repo.ubports.com
-================
+----------------
+
+This is the legacy package repository for UBports projects.  It hosts the Debian packages of the current stable release.
```

Comment:
I'm not a big fan of calling stuff legacy. Especially UBports project has been through so many rounds of calling stuff legacy just to go back on in, or struggling with naming stuff that was three legacies before. 

And even moreso in this particular case. If repo is stable and it will keep being used for stable then it won't be legacy, right? if on the other hand repo is for xenial and repo2 for focal, then repo will quickly not be stable anymore ... can we find an explanation that avoids the word legacy and is somewhat timeless?

Reply:
Well it is fed by the "legacy" build system which follows different conventions. I don't know what the plans for after releasing focal and discontinuing xenial are, i.e. whether repo2 contents will replace repo. I'll leave this open until it has been reviewed by Ratchnanan or Dalton.